README.md

```diff
@@ -13,82 +14,80 @@ http://www.jinteki.net
 
 ## Card implementation status
 
-
 [Card rules implementation status](https://docs.google.com/spreadsheets/d/1ICv19cNjSaW9C-DoEEGH3iFt09PBTob4CAutGex0gnE/pubhtml)
 
 
-## Dependencies
+## Development
+### Quickstart
+
+Install [Leiningen](https://leiningen.org/) and [NodeJS](https://nodejs.org/en/download/package-manager/#macos).
+
+This project runs on Java 8. If you're on OSX or Linux, we recommend using
+[jenv](https://github.com/jenv/jenv/blob/master/README.md) to manage your java environment.
+
+You can check your setup by running
+
+    $ lein version # Your exact version numbers below may vary, but we expect Java 1.8.X
+    Leiningen 2.9.1 on Java 1.8.0_222 OpenJDK 64-Bit Server VM
+
+A quick way to get a MongoDB database set up is using [Docker](https://www.docker.com/):
```

Comment:
I don't feel comfortable jumping directly to Docker in this case. Seems like an oversized solution for our needs. Until we can Docker-ize the whole app, I think it's best to keep the recommendation for "just install mongo and run it in a tab".

Reply:
No worries, I prefer docker for local dev with databases most of the time, since it's a bit easier to manage multiple versions if you need to etc. I'll update this to link back out to mongodb install instructions instead.